src/coreclr/src/vm/methodtable.h

```diff
@@ -2751,22 +2756,17 @@ class MethodTable
         _ASSERTE(GetFlag(enum_flag_Category_Mask) == enum_flag_Category_ValueType);
         SetFlag(enum_flag_Category_Nullable);
     }
-
-    inline BOOL IsStructMarshalable()
-    {
-        LIMITED_METHOD_CONTRACT;
-        PRECONDITION(!IsInterface());
-        return GetFlag(enum_flag_IfNotInterfaceThenMarshalable);
-    }
-
-    inline void SetStructMarshalable()
-    {
-        LIMITED_METHOD_CONTRACT;
-        PRECONDITION(!IsInterface());
-        SetFlag(enum_flag_IfNotInterfaceThenMarshalable);
-    }
     
-    // The following methods are only valid for the method tables for array types.  
+    // The following methods are only valid for the 
+    // method tables for array types.  These MTs may 
```

Comment:
This looks like incorrectly resolved merge conflict. I believe Vlad deleted this comment in his change.